packages/lib-core/src/types/redux.ts

```diff
@@ -0,0 +1,27 @@
+import type { Map as ImmutableMap } from 'immutable';
+import type { K8sResourceCommon } from './k8s';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type K8sState = ImmutableMap<string, any>;
```

Comment:
`any` types are bad... we shouldn't have this. Use `AnyObject` ([link](https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/common/src/types/common.ts#L8)) for now and put a TODO for us to expand it out when we start putting the reducer together.